src/Controller/Controller.php

```diff
@@ -159,7 +163,7 @@ class Controller implements EventListenerInterface, EventDispatcherInterface
      *
      * @var bool
      */
-    public $autoRender = true;
+    protected $autoRender = true;
```

Comment:
Subclasses could call `$this->enable/disableAutoRender()` in their initialize methods.